Mixpanel/Mixpanel+HostWatchOS.m

```diff
@@ -14,7 +14,7 @@ @implementation Mixpanel (HostWatchOS)
 - (void)session:(WCSession *)session didReceiveMessage:(NSDictionary<NSString *, id> *)message {
     NSString *messageType = [Mixpanel messageTypeForWatchSessionMessage:message];
     if (messageType) {
-        if ([messageType isEqualToString:@"track"]) {
+        if (NSStringFromSelector(@selector(track:properties:))) {
```

Comment:
Sorry, should have been more clear, we want 

`[messageType isEqualToString:NSStringFromSelector(@selector(track:properties:))]`